lib/protractor.js

```diff
@@ -110,6 +110,13 @@ var Protractor = function(webdriverInstance, opt_baseUrl, opt_rootElement) {
   this.driver = webdriverInstance;
 
   /**
+   * When running a test suite, this variable should have the name of the suite.
+   *
+   * @type {String}
+   */
+  this.currentSuite = undefined;
```

Comment:
@hankduan Sorry, the line that sets that variable is on minijasminenode. It seems I didn't commit that because it's ignored by .gitignore. I've just realized that it's a different project, so this solution is flawed.